### ISSUE TITLE
Can't Load Spaces

### ISSUE THREAD

**Issue by FarukOzderim** [gradio-app/gradio#451]
**Describe the bug**
Cannot load an example [space](spaces/Norod78/ComicsHeroHD)
**To Reproduce**
```
import gradio as gr
if2 = gr.Interface.load("spaces/Norod78/ComicsHeroHD")
```
**Device information (please complete the following information):**
 - Google Colab
 - Gradio version [2.7.0]  

**Additional context**
Error Message:
```
----> 4 if2 = gr.Interface.load("spaces/Norod78/ComicsHeroHD")

2 frames

/usr/local/lib/python3.7/dist-packages/gradio/external.py in get_spaces_interface(model_name, api_key, alias)
    213     r = requests.get(iframe_url)
    214     result = re.search('window.config =(.*?);\n', r.text) # some basic regex to extract the config
--> 215     config = json.loads(result.group(1))
    216     interface_info = interface_params_from_config(config)
    217 

AttributeError: 'NoneType' object has no attribute 'group'


```

**Comment by abidlabs** [comment]
Confirming I can reproduce the issue. Not sure why the regex doesn't work -- will look into this.

**Comment by abidlabs** [comment]
Issue resolved as part of the latest changes in #440

**Comment by abidlabs** [comment]
Fixed in #440!
